spec/lib/tasks/scripts/add_permission_to_access_level_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'rails_helper'
+require 'tasks/scripts/add_permission_to_access_level'
+
+RSpec.describe AddPermissionToAccessLevel do
+  let!(:organization) { create(:organization) }
+  let!(:facility_group_1) { create(:facility_group, organization: organization) }
+  let!(:facility_group_2) { create(:facility_group, organization: organization) }
+  let!(:owner) { create(:admin, :owner) }
+  let!(:organization_owner) { create(:admin, :organization_owner, organization: organization) }
+  let!(:supervisor_1) { create(:admin, :supervisor, facility_group: facility_group_1) }
+  let!(:supervisor_2) { create(:admin, :supervisor, facility_group: facility_group_2) }
+  let!(:access_levels) { Permissions::ACCESS_LEVELS }
+  let!(:all_permissions) { Permissions::ALL_PERMISSIONS }
+
+  before do
+    create(:user_permission, user: supervisor_1, permission_slug: :manage_facilities, resource_type: 'FacilityGroup', resource_id: facility_group_2.id)
+  end
```

Comment:
Can we move setup data to the section of the specs that it's used in? For example, which parts of this is only relevant to the `create` section, and not the `valid` section?